components/UserDropdownMenu/UserDropdownMenu.jsx

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+require('./UserDropdownMenu.scss')
+
+const React = require('react')
+const Dropdown = require('../Dropdown/Dropdown.jsx')
+const userDropdownLists = [
+  [
+    { label: 'My Profile', link: 'javascript:;', id: 0 },
+    { label: 'Dashboard', link: 'javascript:;', id: 1 },
+    { label: 'Settings', link: 'javascript:;', id: 2 },
+    { label: 'Payments', link: 'javascript:;', id: 3 }
+  ],
+  [
+    { label: 'Help', link: 'javascript:;', id: 0 }
+  ],
+  [
+    { label: 'Log out', link: 'javascript:;', id: 0 }
+  ]
+]
+
+const UserDropdownMenu = {
+  getInitialState () {
+    return { isLoggedIn: true }
+  },
+  render() {
+    let dom
+
+    if (!this.state.isLoggedIn) {
+      dom = <div><button>Log in</button><button>Join</button></div>
```

Comment:
Can we save these two different values for `dom` as descriptive variables, and then `return this.state.isLoggedIn ? dropdown : accountButtons` (doesn't have to be those exact `var` names